src/clients/openapi.js

```diff
@@ -66,11 +66,13 @@ export function buildHeaders(context, req) {
     return headers;
 }
 
-function validateResponse(response) {
+export function validateResponse(response) {
```

Comment:
I've removed the `throw` part from this function to `http` function - to behave in similar way to other logs / exceptions.